src/MICore/CommandFactories/gdb.cs

```diff
@@ -258,5 +263,17 @@ private static string TypeBySize(uint size)
             }
             return TargetArchitecture.Unknown;
         }
+
+        public override async Task Signal(string sig)
```

Comment:
We have other code paths that do this without an interpreter exec. Consider naming this in a way to call out the difference?